adops/gam-creative-banner-sbs.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: page_v2
+title: GAM Step by Step - Banner/Outstream/AMP Creatives
+head_title: GAM Step by Step - Banner/Outstream/AMP Creatives
+description: Set up banner and outstream creatives for Prebid in Google Ad Manager.
+pid: 2
+sidebarType: 3
+---
+
+
+# GAM Step by Step - Banner/Outstream/AMP Creatives
+
+This page walks you through the steps required to create banner and outstream creatives to attach to your Prebid line items in Google Ad Manager (GAM).
+
+{: .alert.alert-success :}
+For complete instructions on setting up Prebid line items in Google Ad Manager, see [Google Ad Manager with Prebid Step by Step](/adops/step-by-step.html).
+
+
+1. In GAM, select **Delivery** > **Creatives**.
+2. Under the **Display creatives** tab, click **New Creative**.
+3. Select your advertiser, then click **Third party**.
+4. Enter a **Name** for your creative. For example, `Prebid – banner – 1x1`.
+5. Enter a **Target ad unit size** of `1x1`. This allows the creative to serve on all inventory sizes.
+
+{: .alert.alert-info :}
+These instructions assume you're using the Prebid Universal Creative (PUC). See the [Prebid Universal Creative](/overview/prebid-universal-creative.html) documentation for alternate approaches.
+
+{:start="6"}
+6. Select **Standard** as the **Code type**.
+
+{: .alert.alert-info :}
+**AMP**: If you are using AMP, you should still select **Standard** as the code type. The “AMP” option is for AMPHTML hosted by a 3rd party.
```

Comment:
This came up recently in slack, so suggest adding "If you choose to bypass the PUC for AMP, Prebid Server events will not work."